spec/models/rent_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Rent, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
We could add some specs to validate the model's validations (with shoulda matchers)

Reply:
I didn't correct this file becouse to test this class i need to incorpore previous PRs